src/hotspot/share/opto/escape.hpp

```diff
@@ -592,8 +592,8 @@ class ConnectionGraph: public ArenaObj {
   // Methods related to Reduce Allocation Merges
   bool has_non_reducible_merge(FieldNode* field, Unique_Node_List& reducible_merges);
   PhiNode* create_selector(PhiNode* ophi) const;
-  void updates_after_load_split(Node* data_phi, Node* previous_load, GrowableArray<Node *>  &alloc_worklist);
-  Node* split_castpp_load_through_phi(Node* curr_addp, Node* curr_load, Node* region, GrowableArray<Node*>* bases_for_loads, GrowableArray<Node *>  &alloc_worklist);
+  void updates_after_load_split(Node* data_phi, Node* previous_load, GrowableArray<Node *>  &alloc_worklist, Unique_Node_List &reducible_merges);
+  Node* split_castpp_load_through_phi(Node* curr_addp, Node* curr_load, Node* region, GrowableArray<Node*>* bases_for_loads, GrowableArray<Node *>  &alloc_worklist,  Unique_Node_List &reducible_merges);
```

Comment:
nit: too many spaces before Unique_Node_List (not sure if it's a required style issue, because the &alloc_worklist parameter also seems to have extra spaces before it....)

Reply:
Removed extra space (Only one space is required).